individual.py

```diff
@@ -48,12 +59,12 @@ def __init__(self, **kwargs):
         self.strength = kwargs.get('strength') or random.choice(range(20, 50, 5))
         self.is_pregnent = False
         self.feminity = random.choice(range(0, 100, 5))  # NOTE(Manish): determins sex of off-spring
-        super().__init__(**kwargs)
+        super().__init__(gender=Gender.FEMALE, **kwargs)
```

Comment:
I think Gender attribute should be part of Individual. Both Female n Male can inherit it. Else we repeat the same code in two places no? Any advantage of having them in child level constructor?

Reply:
A Male object will always have gender Gender.MALE, and a Female object will always have gender Gender.FEMALE. That information is universally known, we should not need to further tell the program that a Male is of MALE gender type, unless it is intended for other purpose as well(like LGBTQ). Even if we account for LGBTQ, we will only end up having 2 sexes relevant to a procreation process in our use-case. Having to tell each time that a Male object is of MALE gender type is redundant, and discards the core essence of inheritance. Gender attribute is still a part of Individual, we have the power to pre-assign it to the correct type since we are using inheritance